CP-CPS.md

```diff
@@ -0,0 +1,1440 @@
+  * Applicable law and the legal requirements for identification of a party, protection of the confidentiality or privacy of information, and enforceability of the transaction;
+  * The intended use of the certificate as listed in the certificate or this CP/CPS,
+  * The data listed in the certificate,
+  * The economic value of the transaction or communication,
+  * The potential loss or damage that would be caused by an erroneous identification or a loss of confidentiality or privacy of information in the application, transaction, or communication,
+  * The Relying Party's previous course of dealing with the Subscriber,
+  * The Relying Party's understanding of trade, including experience with computer-based methods of trade, and
+  * Any other indicia of reliability or unreliability pertaining to the Subscriber and/or the application, communication, or transaction.
+
+Any unauthorized reliance on a certificate is at a party's own risk.
+
+### 9.6.5 Representations and warranties of other participants
+
+No stipulation.
+
+## 9.7 Disclaimers of warranties
+
+ISRG certificates and services are provided "as-is." ISRG disclaims any and all warranties of any type, whether express or implied, including and without limitation any implied warranty of title, non-infringement, merchantability, or fitness for a particular purpose, in connection with any ISRG service or ISRG certificate.
+
+## 9.8 Limitations of liability
+
+ISRG does not accept any liability for any loss, harm, claim, or attorney's fees in connection with any certificates. ISRG will not be liable for any damages, attorney's fees, or recovery, regardless of whether such damages are direct, consequential, indirect, incidental, special, exemplary, punitive, or compensatory, even if ISRG has been advised of the possibility of such damages. This limitation on liability applies irrespective of the theory of liability, i.e., whether the theory of liability is based upon contract, warranty, indemnification, contribution, tort, equity, statute or regulation, common law, or any other source of law, standard of care, category of claim, notion of fault or responsibility, or theory of recovery. This disclaimer is intended to be construed to the fullest extent allowed by applicable law.
+
+Without waiving or limiting the foregoing in any way, ISRG does not make, and ISRG expressly disclaims, any warranty regarding its right to use any technology, invention, technical design, process, or business method used in either issuing certificates or providing any of ISRG's services. Each Subscriber affirmatively and expressly waives the right to hold ISRG responsible in any way, or seek indemnification against ISRG, for any infringement of intellectual property rights, including patent, trademark, trade secret, or copyright.
+
+## 9.9 Indemnities
+
+### 9.9.1 Indemnification by ISRG
+
+The CA does not provide any indemnification except as described in Section 9.9.1 of the Certificate Policy.
```

Comment:
This seems like it needs to be updated -- now that this is a combined document, this _is_ Section 9.9.1 of the CP, and this is self-referential.

Reply:
Good catch! Fixed.